src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -75,19 +64,14 @@ public CallbackService(final SecurityUtils securityUtils,
                                                                     caseEvent.getId(),
                                                                     ignoreWarning);
         final List<Integer> retries = CollectionUtils.isEmpty(callbackRetries) ? defaultRetries : callbackRetries;
-
-        for (Integer timeout : retries) {
-            final Optional<ResponseEntity<CallbackResponse>> responseEntity = sendRequest(url,
-                                                                                          CallbackResponse.class,
-                                                                                          callbackRequest,
-                                                                                          timeout);
-            if (responseEntity.isPresent()) {
-                return Optional.of(responseEntity.get().getBody());
-            }
+        final Optional<ResponseEntity<CallbackResponse>> responseEntity = sendRequest(url, CallbackResponse.class, callbackRequest);
+        if (responseEntity.isPresent()) {
+            return Optional.of(responseEntity.get().getBody());
         }
-        throw new CallbackException("Unsuccessful callback to " + url);
+    throw new CallbackException("Unsuccessful callback to " + url);
```

Comment:
indentation seems wrong to me

Reply:
reformatted the whole class, but looks ok